crates/iota-keys/tests/tests.rs

```diff
@@ -163,6 +163,21 @@ fn update_alias_test() {
     let update = keystore.update_alias("o", None).unwrap();
     let aliases = keystore.alias_names();
     assert_eq!(vec![&update], aliases);
+
+    // check that updating alias does not allow duplicates
+    keystore
+        .generate_and_add_new_key(
+            SignatureScheme::ED25519,
+            Some("my_alias_test".to_string()),
+            None,
+            None,
+        )
+        .unwrap();
+    assert!(
+        keystore
+            .update_alias("my_alias_test", Some(&update))
+            .is_err()
```

Comment:
Can't we match on the exact error? Is there a `DuplicateAliasError` or smth? That'll make fixing the test - should it ever fail - easier I think.

Reply:
No, `bail!` is used so there is no proper typing, and I'd rather not match on the string itself